### ISSUE TITLE
Add tours to explore map mode

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/cac-tripplanner#1137]
Display tour cards with an image carousel.
Highlight all destinations in tour on tour card hover.
Related to modifying the map marker popover to add the optional tour link: #1134.

Depends on #1111 

![image](https://user-images.githubusercontent.com/960264/63878192-16b95480-c997-11e9-996c-1b255872764e.png)

**Comment by flibbertigibbet** [comment]
Closing in favor of #1146 (carousel) and #1121.
